modules/ROOT/pages/project-best-practices-index.adoc

```diff
@@ -1,33 +1,51 @@
 = Project best practices
-:description: Helpful experience we share for the good of your project.
-
-Helpful experience we share for the good of your project.
-
-blabla
-[.card-section]
-== Worth trying
-
-[.card.card-index]
---
-xref:teamwork.adoc[[.card-title]#Create a multi-skilled team# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:lifecycle-and-profiles.adoc[[.card-title]#Project phases# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:agility.adoc[[.card-title]#Agility# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:software-craftsmanship.adoc[[.card-title]#Software craftsmanship# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
-
-[.card.card-index]
---
-xref:devops.adoc[[.card-title]#DevOps# [.card-body.card-content-overflow]#pass:q[TODO]#]
---
+:description: Helpful experience we share for the good of Bonita projects.
+
+Helpful experience we share for the good of Bonita projects.
+
+This section contains two chapters today, but we will keep improving to share more.
+We also encourage you to watch the videos that have been done on this wide topic.
+
+== Teamwork
+To improve the teamwork between business teams who will benefit from the projet, and the IT team that will create it, Bonita has created a xref:project-documentation-generation.adoc[project documentation feature] (Subscription editions only) that captures the content of the project, the business needs and the development explanations (written in the descriptions of all designed items).
+
+== Project phases
+
+xref:design-methodology.adoc[In this page], find the 6 project phases we recommend for a successful project.
+
+
+== image:images/tv.png[Videos]
+
+=== A user-centered approach to make the automation project a success
+* https://www.bonitasoft.com/videos/user-centered-design-guarantee-your-business-process-automation-projects-succeed[In English]
```

Comment:
```suggestion
* https://www.bonitasoft.com/videos/user-centered-design-guarantee-your-business-process-automation-projects-succeed[In English 🇬🇧]
```

Reply:
It's rather english US, as we use "z" in "organization" for example. https://en.wikipedia.org/wiki/American_and_British_English_spelling_differences#-ise,_-ize_(-isation,_-ization)